hr_timesheet_day_week/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+{
+    "name": "Add Weekdays to time sheets",
```

Comment:
Better this:
```suggestion
    "name": "Add day of week management to timesheets",
```
* 'Weekday' is different than 'day of week' AFAIK. 'Weekday' only refers to a day that is used to go work.
* 'Time sheet' is ambiguous, better official 'timesheet'.